### PR TITLE
Fixes#11 [No Prefix Option]

### LINKED ISSUE #11: No prefix option
Hey,

### Prefix Option/Parameter

Don't you think, there should be a no prefix option

### Results
`ALGOLIA_PREFIX` not declared in env file, the result of no prefix is "environment_index"  
`ALGOLIA_PREFIX=` the result of empty prefix is "_index" which is annoying

I can make a pull request for it, if you agree
Thank you

### PR DESCRIPTION
**Pull request by kanrio**
### Default Prefix
Changed default prefix to `false`

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ module.exports = strapi => {
       applicationId: '',
       apiKey: '',
       debug: false,
-      prefix: strapi.config.environment,
```
Comment (MattieBelt) [inline lib/index.js]: I don't think `false` should be the default value here. I think it is a best practice to use environments in the index names.
So no need to change it here and in the [README.md](https://github.com/MattieBelt/strapi-hook-algolia/blob/86d0f1e4d610c2959e4a541526fac176981e1ee7/README.md).

File: lib/index.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ module.exports = strapi => {
       const client = algoliasearch(applicationId, apiKey);
 
       const initIndex = (indexName) => {
-        return client.initIndex(`${prefix}_${indexName}`);
+        const index = (prefix && `${prefix}_${indexName}`) || `${indexName}`;
```
Comment (MattieBelt) [inline lib/index.js]: Template string here is redundant.

```suggestion
        const index = (prefix && `${prefix}_${indexName}`) || indexName;
```

As well as `prefix` should be strictly equal to `false`.
Something like this
```js
return client.initIndex((prefix === false) ? indexName : `${prefix}_${indexName}`);
```

File: README.md [hunk 3]
```diff
@@ -128,7 +128,7 @@ To activate and configure the hook, you need to create or update the file `./con
         applicationId: 'ABCDEFGHIJ',
         apiKey: 'secure_algolia_admin_api_key',
         debug: true,              // default: false
-        prefix: 'my_own_prefix'   // default: Strapi environment (strapi.config.environment)
```
Comment (MattieBelt) [inline README.md]: The default value should remain `strapi.config.environment`, but the possibility to disable the prefix with `prefix: false` should be documented somewhere in the README.md.

**Review by MattieBelt — CHANGES_REQUESTED** [review]
Hey @anouarkacem,

Thanks for submitting this PR, I do think some changes are required before it can be merged.

Let me know if you have any question about the review.

**Comment by kanrio** [comment]
hey @MattieBelt 
Yes agree with you, thank you for the review I made the requested changes

File: README.md [hunk 4]
```diff
@@ -133,6 +133,9 @@ To activate and configure the hook, you need to create or update the file `./con
     }
   };
 ```
+### Prefix
+- Default prefix (string): strapi.config.environment (dev_index, prod_index)
+- No prefix: false (index)
```
Comment (MattieBelt) [inline README.md]: I think something like this explains the prefix usage better:

```suggestion
### Prefix
The default prefix uses `strapi.config.environment`, resulting in for example: 'development_post'.
A custom prefix can be configured with the `prefix` option in the hook config.   
If you don't want to use any prefix at all, you can configure it with: `prefix: false`.
```

**Review by MattieBelt — CHANGES_REQUESTED** [review]
The `index.js` looks fine, just a small change on the docs.

**Review by MattieBelt — APPROVED** [review]

**Comment by MattieBelt** [comment]
LGTM, thanks again!

**Comment by kanrio** [comment]
Great, thank you too !
What about releasing ?

**Pull request merged**